src/data/list/zip.lean

```diff
@@ -242,4 +264,37 @@ begin
         simp * at *, }, }, },
 end
 
+@[simp] lemma map_uncurry_zip_eq_zip_with
+  (f : α → β → γ) (l : list α) (l' : list β) :
+  map (function.uncurry f) (l.zip l') = zip_with f l l' :=
```

Comment:
It's not clear to me which way around this equation should go: is `map ... zip` simpler than `zip_with` or the other way around? I would therefore not make this a `simp` lemma.

Reply:
simp removed